static/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -279,9 +279,7 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
       </Fragment>
     );
 
-    const hasQuickTraceView =
-      organization.features.includes('trace-view-quick') ||
-      organization.features.includes('trace-view-summary');
+    const hasQuickTraceView = organization.features.includes('performance-view');
```

Comment:
This still locks the new layout behind the `performance-view` flag until the appropriate empty states for discover are in place.